services/director-v2/src/simcore_service_director_v2/models/dynamic_services_scheduler.py

```diff
@@ -426,6 +439,16 @@ def get_proxy_endpoint(self) -> AnyHttpUrl:
         "If set to None, the current product is undefined. Mostly for backwards compatibility",
     )
 
+    @root_validator(pre=True)
+    @classmethod
+    def _wallet_info_legacy_migration(cls, values):
+        logger.warning(
+            "check notes for deprecation at https://github.com/ITISFoundation/osparc-simcore/issues/4745"
```

Comment:
please open your issue here and update the link.
this one is pointing to one of my own